pkgs/tools/misc/trashy/default.nix

```
@@ -17,7 +17,10 @@ rustPlatform.buildRustPackage rec {
     installShellCompletion --cmd trash \
       --bash <($out/bin/trash completions bash) \
       --fish <($out/bin/trash completions fish) \
-      --zsh <($out/bin/trash completions zsh) \
+      --zsh <($out/bin/trash completions zsh)
+    $out/bin/trash manpage > trash.1
+    cp trash.1 trashy.1
+    installManPage trash.1 trashy.1
```

Considering https://github.com/oberblastmeister/trashy/pull/106 is merged, will this work?
```suggestion
    $out/bin/trash manpage > trashy.1
    installManPage trashy.1
```

---

Nice catch! I meant to comment on this PR about that trashy PR getting merged. Maybe we should just wait until trashy gets a new release? Is it normal to do an unstable version bump for a package that has a release cadence?

---

If the release will take too long, then consider this in the meanwhile 

```suggestion
    $out/bin/trash manpage > trash.1
    installManPage trash.1
```